relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -193,51 +201,99 @@ fn scrub_file(description: &str) -> Option<String> {
     }
 }
 
-/// Normalize the domain and port of a URL.
+/// Scrub a `Url::Host` object. For domain names, run the string through a scrubber. For IP addresses, return the IP address as-is.
+///
+/// # Arguments
+///
+/// * `host` - `Url::Host`
+///
+/// # Returns
+///
+/// A host `String`, or `None` if scrubbing fails.
```

Comment:
We don't usually document arguments like here, but I wouldn't mind if it provided actual additional information which can't already be gained from the type system.

You'd gain more from adding an example instead (which would also double down as a unittest).

I'd use something like this:

```suggestion
/// Scrub a [`Host`] object. 
///
/// Domain names are run through a scrubber. All IP addresses except well known ones are replaced with a scrubbed variant.
/// Returns the scrubbed value as a string.
///
/// # Examples
///
/// ```
/// assert_eq!(&scrub_host(Host::Domain("foo.bar.baz"), "expected value"));
/// assert_eq!(&scrub_host(Host::Ipv4(Ipv4Addr::LOCALHOST), "127.0.0.1"));
/// ...
/// ```
```